index.js

```diff
@@ -66,6 +67,12 @@ module.exports = {
     }
   },
 
+  validateDependencies() {
+    if ('bootstrap' in this.app.project.bowerDependencies()) {
```

Comment:
We also remove `bootstrap-sass` from bower.json unconditionally and install it in package.json. No dependencies should be in bower.json once they've upgraded.